### PR TITLE
[MIG] : Migrated mrp_bom_location to v13.

### PR DESCRIPTION
**Pull request by sudhir-erpharbor**

### CODE REVIEW

**Review by LoisRForgeFlow — CHANGES_REQUESTED** [review]
Thanks for your contribution, see my comments below:

The columns are to tight, see:

image

File: mrp_bom_location/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "MRP BOM Location",
+    "summary": "Adds location field to Bill of Materials and its components.",
+    "version": "13.0.1.0.1",
+    "category": "Manufacture",
+    "website": "https://github.com/OCA/manufacture",
+    "author": "Eficent, Odoo Community Association (OCA)",
```
Comment (AaronHForgeFlow) [inline mrp_bom_location/__manifest__.py:9]: Could you change Eficent by ForgeFlow? Thank you!

Comment (sudhir-erpharbor) [reply mrp_bom_location/__manifest__.py:9]: I can but it is really a valid change as we will be changing the author.

Comment (LoisRForgeFlow) [reply mrp_bom_location/__manifest__.py:9]: Yes, it is. Our company has been rebranded: https://www.forgeflow.com/blog/our-blog-1/post/from-eficent-to-forgeflow-2

Comment (LoisRForgeFlow) [reply mrp_bom_location/__manifest__.py:9]: But I think we can merge like this and we can open a new PR to change the copyrights, so no worries.

**Comment by LoisRForgeFlow** [comment]
@sudhir-erpharbor can you check the CI result?, something went wrong. Thanks

**Comment by sudhir-erpharbor** [comment]
@LoisRForgeFlow  Said changes are done and travis is green now.

**Comment by LoisRForgeFlow** [comment]
@sudhir-erpharbor CI includes travis AND runbot. Runbot was still red, I just fixed it, hope you don't mind, I wanted to speed up this a little bit.

/ocabot merge

**Review by LoisRForgeFlow — APPROVED** [review]

**Comment by LoisRForgeFlow** [comment]
Seems the merge is stuck.

/ocabot merge

**Pull request merged**